### PR TITLE
Fix accepted doc approval map view initialization [#184315076]

### PR DESCRIPTION
**Pull request by olzraiti**
https://www.pivotaltracker.com/story/show/184315076
https://dev.laji.fi/project/MHL.27/submissions/JX.295185

Fixes two things for accepted document approval view:
* null pointer error in the template code when form permissions badges are rendered
* Map view initialization - not sure what had broke but I made it work

### CODE REVIEW

File: projects/laji/src/app/+project-form/project-form.component.html [hunk 1]
```diff
@@ -69,11 +69,13 @@ <h5>
 </ng-template>
 
 <ng-template #badge let-badge>
+  <ng-container *ngIf="formPermissions$ | async as formPermissions">
       <span
-        *ngIf="(formPermissions$ | async)[badge.count]"
+        *ngIf="formPermissions$[badge.count]"
```
Comment (aorin) [inline projects/laji/src/app/+project-form/project-form.component.html:74]: Shouldn't this be `formPermissions` instead of `formPermissions$`?

**Review by aorin — CHANGES_REQUESTED** [review]

File: projects/laji/src/app/+project-form/submissions/statistics/common/accepted-document-approval/accepted-document-approval.component.html [hunk 2]
```diff
@@ -29,6 +29,6 @@
           (click)="acceptNamedPlaceChanges()"
           class="btn btn-sm btn-success ml-2">Hyväksy muutokset</button>
 </div>
-<div class="col-xs-12" [style.height.px]="1000" [style.display]="isAdmin && geometriesDiff ? 'block' : 'none'">
+<div *ngIf="isAdmin" class="col-xs-12" [style.height.px]="1000">
```
Comment (aorin) [inline projects/laji/src/app/+project-form/submissions/statistics/common/accepted-document-approval/accepted-document-approval.component.html:32]: Just to make sure, is it intentional that this element is also shown when `geometriesDiff` is `undefined`?

File: projects/laji/src/app/+project-form/project-form.component.html (hunk 1 above)
Comment (olzraiti) [reply projects/laji/src/app/+project-form/project-form.component.html]: Yes - fixed

**Review by aorin — APPROVED** [review]
:+1:

**Pull request merged**